### DISCUSSION TITLE
Stat Panel: Ability to hide fields but still reference them in data links

### DISCUSSION THREAD

**Discussion by raj-shah14** [grafana/grafana#68274] [Dashboards, Panels & Visualizations]
![image](https://github.com/grafana/grafana/assets/25533503/7b1948ba-6e7a-4bcc-bb28-f98c21d438c4)
I am able to hide the Guid field using Transform but not able to access that field in data link using ${__data.fields.RolloutId}

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82675. Feel free to continue the discussion around this there. Thank you!
